test/type/string_test.exs

```diff
@@ -0,0 +1,107 @@
+defmodule Litmus.Type.StringTest do
+  use ExUnit.Case
+
+  alias Litmus.Type
+
+  test "tests protocol implementation for String data type" do
+    type = %Type.String{
+      required: true
+    }
+
+    field = :id
+
+    data = %{
+      id: "1"
+    }
+
+    assert Type.validate(type, field, data) == {:ok, data}
+  end
+
+  test "tests validate_keys function for String data type" do
+    type = %Type.String{
+      required: true
+    }
+
+    field = :id
+
+    data = %{
+      id: "1"
```

Comment:
i believe all query and body params come into Dora (Plug) with string fields, not atoms. Should we change our tests to be more representative of actual usage? @shraddha1704 @kylekwong

Reply:
Sure. Should we have a test for both cases? string and atoms?